### DISCUSSION TITLE
Firebase file - where do I get it from

### DISCUSSION THREAD

**Discussion by freerbeing** [nunocoracao/blowfish#802] [General]
I have established a Firebase account
I dont see all the config detilas I need to add to params/toml

In the doc page for Firebase you say
"You can find an example of the file Firebase will provide below, "

Where does Firebase provide this file, please?
 .. I have looked, honest!

I have some details but cannot yet see ...

   storageBucket = "XXXXXX"
   messagingSenderId = "XXXXXX"
   measurementId = "XXXXXX"

**Comment by nunocoracao** [comment]
Try to add a new web project, and follow that journey until the page displays a JSON code snipped. That snipped will include all the required values.

**Comment by freerbeing** [comment]
Thanks, buddy
Actually I find that is works fine with just the following
 [firebase]
   apiKey = " xxx"
   authDomain = "xxx"
  projectId = "xxx"
  appId = "xxx"

I got these easily from the Firebase account details page
